### ISSUE TITLE
Integer EMC causes issues with small items

### ISSUE THREAD

**Issue by AngleWyrm10** [sinkillerj/ProjectE#2229]
ProjectE v1.18.2-PE1.0.1
Forge version: 40.1.73

Steps to reproduce:
Six 1 EMC glass blocks = 16 glass panes -- how much EMC is a glass pane worth?

What I expected to happen:
6 EMC / 16 panes = 3 EMC per 8 panes = 0.375 EMC/pane

What happened instead:
glass pane EMC unlisted

This is problematic when such items are recipe ingredients, or when generating stacks of them.

**Comment by GhostShot3** [comment]
Yep, I agree, and this happens with a lot of types of small EMC items. EMC should use a long type with decimals or something similar
